bin/tronfig

```diff
@@ -166,7 +166,11 @@ if __name__ == '__main__':
     client = Client(args.server)
 
     if args.print_config:
-        print(client.config(args.source)['config'])
+        # TODO: use maybe_encode()
```

Comment:
Can we not just .encode('utf8') all the time?

Reply:
that would break python3 compatibility, i'll clean it all up after we upgrade